android/app/src/main/res/layout/fragment_about.xml

```diff
@@ -1,7 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout
```

Comment:
What's the use of RelativeLayout here

Reply:
relative layout is used to keep social link recyclerview at parent bottom